Makefile

```diff
@@ -133,9 +134,48 @@ build-darwin:
 	  $(MAKE) build
 
 build-linux:
-	SC_GOOS=linux SC_GOARCH=amd64 BINDIR=bin/darwin_amd64 DOCKER=1 \
+	SC_GOOS=linux SC_GOARCH=amd64 BINDIR=bin/linux_amd64 DOCKER=1 \
 	  $(MAKE) build
 
+images: build-linux $(shell find pkg contrib -name Dockerfile)
```

Comment:
So I can't build _an_ image? I have to build them all? And that in turn requires me to build _all_ the binaries... that doesn't sit well with me since I have so little interest, for instance, in what is in contrib. imo, it would be much much better if I at least had the option to build a specific image.